lib/path.js

```diff
@@ -669,10 +669,10 @@ const win32 = {
     if (ext !== undefined)
       validateString(ext, 'ext');
     validateString(path, 'path');
-    var start = 0;
-    var end = -1;
-    var matchedSlash = true;
-    var i;
+    let start = 0;
+    let end = -1;
+    let matchedSlash = true;
+    let i;
```

Comment:
nit: this `let i` can be removef and moved to lines 691 and 731 respectively

Line 691:
```javascript
for (let i = path.length - 1; i >= start; --i) {
```

Line 731:
```javascript
for (let i = path.length - 1; i >= start; --i) {
```

Reply:
good idea ,i've fixed it;